includes/Classifai/Providers/Azure/TextToSpeech.php

```diff
@@ -436,44 +437,86 @@ private function get_default_settings() {
 			'voices'        => array(),
 			'voice'         => '',
 			'authenticated' => false,
-			'post-types'    => array(),
+			'post_types'    => array(),
 		];
 	}
 
 	/**
-	 * Add `classifai_synthesize_speech` to rest API for view/edit.
+	 * Initial audio generation state.
+	 *
+	 * Fetch the initial state of audio generation prior to the audio existing for the post.
+	 *
+	 * @param  {int|WP_Post|null} $post   Optional. Post ID or post object. `null`, `false`, `0` and other PHP falsey values
+	 *                                    return the current global post inside the loop. A numerically valid post ID that
+	 *                                    points to a non-existent post returns `null`. Defaults to global $post.
+	 * @return {bool}                     The initial state of audio generation. Default true.
+	 */
+	public function get_audio_generation_initial_state( $post = null ) {
+		return apply_filters( 'classifai_audio_generation_initial_state', true, get_post( $post ) );
+	}
+
+	/**
+	 * Subsequent audio generation state.
+	 *
+	 * Fetch the subsequent state of audio generation once audio is generated for the post.
+	 *
+	 * @param {int|WP_Post|null} $post   Optional. Post ID or post object. `null`, `false`, `0` and other PHP falsey values
+	 *                                   return the current global post inside the loop. A numerically valid post ID that
+	 *                                   points to a non-existent post returns `null`. Defaults to global $post.
+	 * @return {bool}                    The subsequenet state of audio generation. Default false.
+	 */
+	public function get_audio_generation_subsequent_state( $post = null ) {
+		return apply_filters( 'classifai_audio_generation_subsequent_state', false, get_post( $post ) );
```

Comment:
For these filters, can we add filter docblocks so these make it to our automated documentation? See https://github.com/10up/classifai/blob/develop/includes/Classifai/Providers/Azure/TextToSpeech.php#L629 as an example

Reply:
Added the doc blocks. But let me know if the wording needs to be updated for it.